src/cpp/CMakeLists.txt

```diff
@@ -64,6 +64,9 @@ if(ENABLE_GGUF)
     add_library(gguflib STATIC ${gguflib_SOURCE_DIR}/fp16.c ${gguflib_SOURCE_DIR}/gguflib.c)
     set_target_properties(gguflib PROPERTIES POSITION_INDEPENDENT_CODE ON)
     target_include_directories(gguflib PUBLIC "${gguflib_SOURCE_DIR}")
+    if(MSVC)
+        target_compile_options(gguf-tools PRIVATE "/sdl-")
```

Comment:
probably, it's better to add `EXCLUDE_FROM_ALL` for GGUF folder

in this case, only `gguflib` will be built

Reply:
Fixed